src/components/ChatLog.jsx

```diff
@@ -0,0 +1,28 @@
+import './ChatLog.css';
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+
+const ChatLog = ({ entries, onToggleLike }) => {
+  return (
+    <section className="chat-log">
+      {entries.map(entry => (
+        <ChatEntry
+          key={entry.id}
```

Comment:
Nice use of the `entry.id` for the `key` values since our messages have unique ids!